packages/fhi-designsystem/src/components/fhi-button/fhi-button.ts

```diff
@@ -0,0 +1,388 @@
+import { html, css, LitElement } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+export const FhiButtonSelector = 'fhi-button';
+
+export type FhiButtonProps = Pick<
+  FhiButton,
+  'color' | 'variant' | 'disabled' | 'size'
+>;
+
+/**
+ * Button component
+ *
+ * @tag fhi-button
+ *
+ * @slot contains the content of the button
+ *
+ */
+@customElement(FhiButtonSelector)
+export class FhiButton extends LitElement {
+  /**
+   * Bestemmer fargen på knappen.
+   * @default 'accent'
+   */
+  @property({ type: String, reflect: true }) color:
+    | 'accent'
+    | 'neutral'
+    | 'danger' = 'accent';
+
+  /**
+   * Bestemmer varianten av knappen.
+   * @default 'strong'
+   */
+  @property({ type: String, reflect: true }) variant:
+    | 'strong'
+    | 'subtle'
+    | 'outlined'
+    | 'text' = 'strong';
+
+  /**
+   * Bestemmer størrelsen på knappen.
+   * @default 'medium'
+   */
+  @property({ type: String, reflect: true }) size?:
+    | 'large'
+    | 'medium'
+    | 'small' = 'medium';
+
+  /**
+   * Bestemmer om knappen er tilgjengelig.
+   * @deafult false
+   */
+  @property({ type: Boolean, reflect: true }) disabled = false;
+
+  render() {
+    return html`<button ?disabled=${this.disabled}>
+      <slot></slot>
+    </button>`;
+  }
+
+  static styles = css`
+    :host {
+      button {
+        border-radius: var(--fhi-border-radius-full);
+        border-style: inset;
+        border: 1px solid;
+        font-family: var(--fhi-font-family-roboto-flex);
+
+        display: inline-flex;
+        justify-content: center;
+        align-items: center;
+        transition-duration: var(--fhi-duration-slow);
+
+        cursor: pointer;
+        &:disabled {
```

Comment:
Vi bør kanskje bli enige om noen syntax-regler for css; linjeskift, rekkefølge osv... bare et innspill til diskusjon 🙂